test/functional/tests/incremental_load/test_incremental_load.py

```diff
@@ -636,31 +637,53 @@ def test_remove_inactive_devices():
     with TestRun.step("Verify that all previously created CAS devices are listed with "
                       "proper status."):
         for core in cores:
-            if core.get_status() != CoreStatus.inactive:
+            if core.get_status(core_dev_id) != CoreStatus.inactive:
```

Comment:
I wonder whether `.get_status(core_id)` should be a method of `Cache` class instead of `Core`. Each `Core` instance should be associated with one core device on DUT. With the current implementation you can check status of the core device which is not associated with the current `core` variable.

Reply:
"With the current implementation you can check status of the core device which is not associated with the current core variable." Correct but is that bad? I would say this is harmless and may be useful

Reply:
If the method is to be used like that, then it should be a class/static method. Current implementation is confusing and might lead to errors.